src/main/java/org/reactome/orthoinference/SkipInstanceChecker.java

```diff
@@ -67,6 +57,13 @@ public static boolean checkIfInstanceShouldBeSkipped(GKInstance reactionInst) th
 			logger.info(reactionInst + " is in skipList -- skipping");
 			return true;
 		}
+
+		// If the only TopLevelPathway of a Reaction is 'Disease', then it is skipped.
+		// Otherwise, it is inferred, making sure in cases where a Reaction is also a part of 'Disease' that that Pathway is not inferred.
+		if (org.reactome.orthoinference.InstanceUtilities.onlyInDiseasePathway(reactionInst)) {
```

Comment:
Is the package qualification needed here for the `onlyInDiseasePathway` method?  It becomes a bit hard to see the method call and the condition.  Could it be statically imported?

Reply:
I changed it so that the non-Orthoinference InstanceUtiltiies is statically imported